windows/src/desktop/kmshell/kmshell.dpr

```diff
@@ -200,7 +201,7 @@ begin
         Application.Initialize;
         Application.Title := 'Keyman Configuration';
         Application.CreateForm(TmodWebHttpServer, modWebHttpServer);
-        try
+  try
```

Comment:
Ah, the indentation of `try` again. Seems like I fixed that in a review of a previous PR and now it's back...

```suggestion
        try
```

Reply:
Marc warned on that previous one even though I applied the change it would probably come back. The `.dpr` file is managed by the Delphi IDE and it gets it wrong.

Reply:
I solved that in Keyman Developer by moving the code into another unit and having a one line call:

https://github.com/keymanapp/keyman/blob/ff5a366c457644814385b06339ac6fb04f91ed12/developer/src/tike/tike.dpr#L309-L311

https://github.com/keymanapp/keyman/blob/3a91c7d197b99fc5bb0a82b92172fe55e1d865f7/developer/src/tike/main/Keyman.Developer.System.Main.pas

Reply:
https://github.com/keymanapp/keyman/issues/10234